src/server/shared/SharedDefines.h

```diff
@@ -527,7 +527,7 @@ enum SpellAttr3 : uint32
 // EnumUtils: DESCRIBE THIS
 enum SpellAttr4 : uint32
 {
-    SPELL_ATTR4_NO_CAST_LOG                        = 0x00000001, // TITLE Cannot be resisted
+    SPELL_ATTR4_IGNORE_ALL_RESISTANCES             = 0x00000001, // TITLE Ignores all resistances
```

Comment:
![EnumeratedString](https://github.com/azerothcore/azerothcore-wotlk/assets/38561765/6f2d914b-abd5-4bf1-911a-1afa9d014324)

Reply:
hmm, perhaps we should be using SPELL_ATTR5_NO_PARTIAL_RESISTS instead of that... ?